packages/core/src/jsx-dev-runtime.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as ReactJSXRuntimeDev from 'react/jsx-dev-runtime'
```

Comment:
OOC do you have a particular reason to prefer the namespace import over named ones? Does it work around webpack's `(0, r.jsx)(...)` compile output, or something?

Reply:
FYI while exploring this, I discovered that the namespace import is actually preferable for the reason I suspected (it bundles more efficiently), but only if the runtime is CJS. So it makes no difference for Emotion but it does make a difference for React. I opened https://github.com/facebook/react/issues/20113 to suggest changing the React Babel plugin output to use a namespace import. 🙂